File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/wrapper/multiplayer/WorldClient.java
package rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.multiplayer;

import rbq.wtf.lycoris.client.wrapper.MapEnum;
import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapClass;
import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapperClass;
import rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.world.World;

@WrapperClass(mcpName = "net.minecraft.client.multiplayer.WorldClient", targetMap = MapEnum.VANILLA189)
public class WorldClient extends World {
    @WrapClass(mcpName = "net.minecraft.client.multiplayer.WorldClient", targetMap = MapEnum.VANILLA189)
    public static Class<?> worldClientClass;

    public WorldClient(Object obj) {
        super(obj);
    }
}

File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/wrapper/multiplayer/WorldClient.kt
package rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.multiplayer

import rbq.wtf.lycoris.client.wrapper.MapEnum
import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapClass
import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapperClass
import rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.world.World

@WrapperClass(mcpName = "net.minecraft.client.multiplayer.WorldClient", targetMap = MapEnum.VANILLA189)
class WorldClient(obj: Any) : World(obj) {
    companion object {
        @WrapClass(mcpName = "net.minecraft.client.multiplayer.WorldClient", targetMap = MapEnum.VANILLA189)
        lateinit var worldClientClass: Class<*>
    }
}